client/src/pages/ProfileListing/CoffeCard.jsx

```diff
@@ -0,0 +1,53 @@
+import React from 'react';
+import Card from '@material-ui/core/Card';
+import CardActions from '@material-ui/core/CardActions';
+import CardContent from '@material-ui/core/CardContent';
+import Typography from '@material-ui/core/Typography';
+import { IconButton, CardMedia } from '@material-ui/core';
+import LocationOnIcon from '@mui/icons-material/LocationOn';
+import Rating from '@mui/material/Rating';
+
+const CoffeCard = (props) => {
+  const { avatarUrl, title, description, imageUrl, maintitle, price, country } = props;

```

Comment:
Rather than destructuring here, could we just do it in the function arguments and adding a type of `React.FC` to the `CoffeCard` or the new name when you rename it. Thanks!